### PR TITLE
docs: add package/gluon-ebtables-limit-arp

### LINKED ISSUE #1383: gluon-ebtables-limit-arp: missing documentation for readthedocs
gluon-ebtables-limit-arp: missing documentation for readthedocs

could be taken from here: https://github.com/freifunk-gluon/gluon/blob/master/package/gluon-ebtables-limit-arp/Makefile#L17

### PR DESCRIPTION
**Pull request by mweinelt**
based on package documentation, authored by T_X
https://github.com/freifunk-gluon/gluon/blob/84a6f65f02d7e36a073ba2839712f0c0bb1dda10/package/gluon-ebtables-limit-arp/Makefile#L18-L39

fixes #1383

### CODE REVIEW

**Comment by rubo77** [comment]
Cache should be lowercase

File: docs/package/gluon-ebtables-limit-arp.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+gluon-ebtables-limit-arp
+========================
+
+The *gluon-ebtables-limit-arp* package adds filters to limit the 
+amount of ARP Requests client devices are allowed to send into the
```
Comment (skorpy2009) [inline docs/package/gluon-ebtables-limit-arp.rst]: ARP requests

File: docs/package/gluon-ebtables-limit-arp.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
+gluon-ebtables-limit-arp
+========================
+
+The *gluon-ebtables-limit-arp* package adds filters to limit the 
+amount of ARP Requests client devices are allowed to send into the 
+mesh. 
+
+The limits per client device, identified by its MAC address, are
+6 packets per minute and 1 per second per node in total. 
+A burst of up to 50 ARP Requests is allowed until the rate-limiting
```
Comment (skorpy2009) [inline docs/package/gluon-ebtables-limit-arp.rst]: ARP requests

**Comment by rotanid** [comment]
squashed and merged according to our "merge faster" policy, especially as it's "only" docs which can't break anything ;-)
if someone has enhancements, he/she may open a new PR

**Pull request merged**